src/missing_deps.cc

```diff
@@ -0,0 +1,158 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
IANAL -- but I am very familiar with copyright issues, having worked with them plenty over my career.

Please consider carefully the license header, unless you are a Google employee.

While the Ninja project does have the explicit guideline that all files have "Copyright *year* Google Inc.", this is very much not the correct thing to do for non-Google employees.

If you are not a Google employee, then you are the owner of the copyright for newly written code. In such case, you can't simply say "Google owns this" without some explicit agreement with them to have the copyright transfer.

Reply:
Allow me to clarify my poor wording.

*obviously* you can *say* whatever you want.

I meant "Saying that Google holds the copyright on this code is not factually correct simply because the file says so. The copyright is retained by the actual author in non-"work for hire" situations unless there's an explicit agreement between the author and Google saying that the rights have been transferred".

Reply:
I believe this would be more appropriate to discuss in https://github.com/ninja-build/ninja/pull/1616 that explicitly mentions the licensing, vs in a random change that happens to add a file.

And anyway, the change here is from a corporate email address with a google CLA in place already due to contributions elsewhere; and https://opensource.google.com/docs/releasing/contributions/#copyright doesn't indicate worrying about the copy header is required.

Reply:
No need for you to do anything. I only wanted to make sure you were aware.